crates/astria-proto/proto/astria/sequencer/v1/data.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package astria.sequencer.v1;
+
+import "tendermint/types/types.proto";
+
+// IndexedTransaction represents a sequencer transaction along with the index
+// it was originally in the sequencer block.
+message IndexedTransaction {
+    uint64 block_index = 1; // TODO: this is usize - how to define for variable size?
```

Comment:
do you mean since there will likely be << max uint64 txs in a block, how can we reduce this so that it's the minimum needed size?

Reply:
no, just that usize can be either 32bit or 64bit depending on compilation target. see Janis' comment: https://github.com/astriaorg/astria/pull/12#discussion_r1198927210